ext/ddtrace_profiling_native_extension/extconf.rb

```diff
@@ -122,6 +140,7 @@ def add_compiler_flag(flag)
 
 # If we got here, libddprof is available and loaded
 ENV['PKG_CONFIG_PATH'] = "#{ENV['PKG_CONFIG_PATH']}:#{Libddprof.pkgconfig_folder}"
+Logging.message(" [ddtrace] PKG_CONFIG_PATH set to '#{ENV['PKG_CONFIG_PATH']}'\n")
```

Comment:
I usually favour calling `#inspect` when logging values: when they're strings they'll be automatically wrapped in quotes, and if they happen to be an unexpected value it becomes very obvious (e.g distinguishing `nil` vs empty string)

Just a note, as here it's obviously a string.

Reply:
Yeap, good point and good rule of thumb, fixed in 455a808eadf819f2d88e5eb656c0cde9e01546db